hushline/model.py

```diff
@@ -256,6 +312,12 @@ def pgp_key(self, value: str) -> None:
         else:
             self._pgp_key = encrypt_field(value)
 
+    def is_free_tier(self) -> bool:
```

Comment:
Make these `@property`s?

Reply:
Fixed in 3757b5ec6137201111e00ed8335b73f8e5388721